### PR TITLE
Chibuzo update readme with my name

### PR DESCRIPTION
**Pull request by Domicaren1**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Other (please specify here: <type here>)
  
  
## Description of what this Pull Request does
(type your description here)

## Related Tickets & Documents
- Closes # (add the ticket number here, like #1)
- Related Issues: (enter ticket numbers here)
  
## QA Instructions, Screenshots, Recordings
(In this section, describe how to QA/review your PR. Feel free to use screenshots and/or short videos)
  
## Added/updated tests?
- [x] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [ ] No, and here is why: __explain here__

### CODE REVIEW

**Comment by msarit** [comment]
@annonymousauthority @jjleigh Can we get some approval so we can merge? 🙏🏾

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Check out our [Next.js deployment documentation](https://nextjs.org/docs/deploym
 
 - [Neka Blair](https://github.com/istoga)
 - [Ojoachele Onuh](https://github.com/Achele)
+- Chibuzo Idia
```
Comment (jjleigh) [inline README.md:46]: @Domicaren1 Can you add your GitHub as well

Comment (Domicaren1) [reply README.md]: Done

**Comment by msarit** [comment]
@Domicaren1 One more merge conflict and then you will be good to go

File: README.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ Check out our [Next.js deployment documentation](https://nextjs.org/docs/deploym
 
 - [Neka Blair](https://github.com/istoga)
 - [Ojoachele Onuh](https://github.com/Achele)
+- Chibuzo Idia (https://github.com/Domicaren1)
```
Comment (annonymousauthority) [inline README.md:46]: This wouldn't come out as 
[Chibuzo Idia](https://github.com/Domicaren1)

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
This wouldn't come out as
[Chibuzo Idia](https://github.com/Domicaren1)

File: README.md (hunk 2 above)
Comment (Domicaren1) [reply README.md:46]: I don't understand @annonymousauthority

Comment (msarit) [reply README.md:46]: @Domicaren1 Compare your entry with the others. What is missing from your entry? These name links are in Markdown format.

Others are: [Name](link)

Yours is: Name (link)

Do you see the difference?

Comment (Domicaren1) [reply README.md:46]: yes ma @msarit . I've seen the difference

**Pull request merged**